OptimizelySDKCore/OptimizelySDKCore/OPTLYEventParameterKeys.m

```diff
@@ -21,16 +21,24 @@
 NSString * const OPTLYEventParameterKeysRevision                        = @"revision";
 NSString * const OPTLYEventParameterKeysActivationId                    = @"activationId";
 NSString * const OPTLYEventParameterKeysSessionId                       = @"sessionId";
```

Comment:
Optimizely python-sdk EventParams doesn't mention "session_id" and I do not
see OPTLYEventParameterKeysSessionId acutally being used anywhere in the
objective-c-sdk .  Therefore, I think you can delete this line.

Reply:
OPTLYEventParameterKeysSessionId removed.